.github/workflows/pull-data.yml

```diff
@@ -130,6 +130,12 @@ jobs:
       - name: Export Report
         run: npx ts-node ./src/export-report.ts
 
+      - name: Export Budgets
+          env:
+            ROLLBAR_ACCESS_TOKEN: ${{ secrets.ROLLBAR_ACCESS_TOKEN }}
+            NODE_OPTIONS: "--max-old-space-size=4096"
```

Comment:
Is it necessary the env var `--max-old-space-size=4096` ? the other script has it because it was running out of memory during execution

Reply:
You are right, I was just being paranoid 🙈